extensions/cactus-plugin-htlc-coordinator/src/test/typescript/integration/api-surface.ts

```diff
@@ -0,0 +1,9 @@
+const tap = require("tap");
```

Comment:
@jagpreetsinghsasan Please rename this to have a `.test.ts` ending + refactor to use the `tape-promise` library (see other api-surface.test.ts files for examples)